### DISCUSSION TITLE
How to configure the proto extension?

### DISCUSSION THREAD

**Discussion by alexkirsz** [zed-industries/zed#22308] [Feature Requests]
### Check for existing issues

- [X] Completed

### Describe the bug / provide steps to reproduce it

https://github.com/lasorda/protobuf-language-server supports settings, but I can't seem to find a way to pass them to the language server.

I've tried the following:

```
  "lsp": {
    "protobuf-language-server": {
      "initialization_options": {
        "settings": {
          "additional-proto-dirs": ["xxx"]
        }
      }
    },
```

```
  "lsp": {
    "protobuf-language-server": {
      "settings": {
        "additional-proto-dirs": ["xxx"]
      }
    },
```

None seem to pass any settings to the language server.

### Environment

Zed: v0.165.4 (Zed)
OS: macOS 15.1.1
Memory: 48 GiB
Architecture: aarch64

### If applicable, attach your Zed.log file to this issue.

<details><summary>Zed.log</summary>

```

```
</details>

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
